ui/workItemTabPage.html

```diff
@@ -0,0 +1,98 @@
+﻿<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Work item extension page sample</title>
+</head>
+
+<body style="overflow:auto;">
+    <script src="sdk/scripts/VSS.SDK.js"></script>
+
+    <script>
+
+        VSS.init({ usePlatformScripts: true });
+
+        // Get the ActiveWorkItemService.  This service allows you to get/set fields/links on the 'active' work item (the work item
+        // that currently is displayed in the UI).
+        function getActiveWorkItemService()
+        {
+            var deferred = $.Deferred();
+            
+            VSS.require(["TFS/WorkItemTracking/Services"], function(_WorkItemServices) {
+                _WorkItemServices.ActiveWorkItemService.getService().then(function (service) {
+                    deferred.resolve(service)
+                },
+                function() {
+                    deferred.reject();
+                });
+            });
+
+            return deferred.promise();           
+        }
+
+        // Register a listener for the work item page contribution
+        VSS.register("SamplePageWorkItemListener", function () {
+            return {
+                // Called when the active work item is modified
+                onFieldChanged: function(args) {
+                    $(".events").append($("<div/>").text("onFieldChanged - " + JSON.stringify(args)));
+                },
+                
+                // Called when a new work item is being loaded in the UI
+                onLoaded: function (args) {
+        
+                    getActiveWorkItemService().then(function(service) {            
+                        // Get the current values for a few of the common fields
+                        service.getFieldValues(["System.Id", "System.Title", "System.State", "System.CreatedDate"]).then(
+                            function (value) {
```

Comment:
Please add a comment what the argument value contains. Is it a list of key/value pairs? if so, then please rename the argument to values